care.config.ts

```diff
@@ -48,6 +48,8 @@ const careConfig = {
     .split(",")
     .map((l) => l.trim()),
 
+  defaultEncounterType: env.REACT_DEFAULT_ENCOUNTER_TYPE || "hh",
```

Comment:
you can make type safety at care config level itself
```suggestion
  defaultEncounterType: (env.REACT_DEFAULT_ENCOUNTER_TYPE || "hh") as EncounterClass,
```